Documentation/index.rst

```diff
@@ -27,6 +27,7 @@ Last Updated: |today|
    components/index.rst
    applications/index.rst
    reference/index.rst
+   faq.rst
```

Comment:
please move into `faq/index.rst` so that we can later split into subdocuments as needed

Reply:
Thank you @v01d all issues were fixed!